local/network.go

```diff
@@ -254,10 +267,16 @@ func newNetwork(
 			nodeConfigs = append(nodeConfigs, nodeConfig)
 		}
 	}
-	net.rootDir, err = os.MkdirTemp("", "avalanche-network-runner-*")
-	if err != nil {
-		return nil, err
+
+	if dir == "" {
+		net.rootDir, err = os.MkdirTemp("", "avalanche-network-runner-*")
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		net.rootDir = dir
```

Comment:
Should we check if this dir actually exists, and maybe even, if not, create it (or warn and exit if it doesn't exist)?